sos/report/plugins/pulp.py

```diff
@@ -131,10 +131,18 @@ def setup(self):
         self.add_cmd_output(prun, suggest_filename="pulp-running_tasks")
         self.add_cmd_output(csizes, suggest_filename="mongo-collection_sizes")
         self.add_cmd_output(dbstats, suggest_filename="mongo-db_stats")
-        self.add_cmd_output([
-            "qpid-stat -%s --ssl-certificate=%s -b amqps://localhost:5671" %
-            (opt, self.messaging_cert_file) for opt in "quc"
-        ])
+        self.add_cmd_output(
+            "qpid-stat -c --ssl-certificate=%s -b amqps://localhost:5671" %
+            self.messaging_cert_file,
+            tags="qpid_stat_c")
+        self.add_cmd_output(
+            "qpid-stat -q --ssl-certificate=%s -b amqps://localhost:5671" %
+            self.messaging_cert_file,
+            tags="qpid_stat_q")
+        self.add_cmd_output(
+            "qpid-stat -u --ssl-certificate=%s -b amqps://localhost:5671" %
+            self.messaging_cert_file,
+            tags="qpid_stat_u")
```

Comment:
This can be de-duplicated:

    for opt in "quc":
         self.add_cmd_output(
            f"qpid-stat -{opt} --ssl-certificate={self.messaging_cert_file} -b amqps://localhost:5671",
            tags=f"qpid_stat_{opt}")

(just I think `pep8` would dislike the huge line)

Reply:
Ahh that's a good point, let me work on that change.